framework/infrastructure/gcp/api.py

```diff
@@ -157,6 +157,8 @@ def compute(self, version: str):
             return self._build_from_discovery_v1(api_name, version)
         elif version == "v1alpha":
             return self._build_from_discovery_v1(api_name, "alpha")
+        elif version in ["v1beta", "v1beta1"]:
```

Comment:
Curious why we have both `v1beta` and `v1beta1` here

Reply:
This is from @larry-safran proof of concept https://github.com/grpc/psm-interop/pull/88/files

Reply:
Spoke with Larry, let's make it `version == "v1beta"`

Reply:
Done